packages/opentelemetry-resources/test/Resource.test.ts

```diff
@@ -84,30 +84,29 @@ describe('Resource', () => {
       'custom.number': 42,
       'custom.boolean': true,
     });
-    assert.equal(resource.attributes['custom.string'], 'strvalue');
-    assert.equal(resource.attributes['custom.number'], 42);
-    assert.equal(resource.attributes['custom.boolean'], true);
+    assert.strictEqual(resource.attributes['custom.string'], 'strvalue');
+    assert.strictEqual(resource.attributes['custom.number'], 42);
+    assert.strictEqual(resource.attributes['custom.boolean'], true);
   });
 
   describe('.empty()', () => {
-    it('should return an empty resource', () => {
+    it('should return an empty resource (except required service name)', () => {
       const resource = Resource.empty();
-      assert.equal(Object.entries(resource.attributes), 0);
+      assert.deepStrictEqual(Object.keys(resource.attributes), []);
     });
 
     it('should return the same empty resource', () => {
       assert.strictEqual(Resource.empty(), Resource.empty());
     });
   });
 
-  describe('.createTelemetrySDKResource()', () => {
-    it('should return a telemetry SDK resource', () => {
-      const resource = Resource.createTelemetrySDKResource();
-      assertTelemetrySDKResource(resource, {
-        language: SDK_INFO.LANGUAGE,
-        name: SDK_INFO.NAME,
-        version: SDK_INFO.VERSION,
-      });
+  describe('.default()', () => {
+    it('should return a default resource', () => {
+      const resource = Resource.default();
+      assert.strictEqual(resource.attributes[ResourceAttributes.TELEMETRY_SDK_NAME], SDK_INFO[ResourceAttributes.TELEMETRY_SDK_NAME]);
```

Comment:
are we allowing the long lines or the rule for that was accidently removed ?

Reply:
I think it was a prettier rule. We can add it back.